core/src/main/java/dev/streamx/cli/config/DotStreamxGeneratedConfigSource.java

```diff
@@ -0,0 +1,42 @@
+package dev.streamx.cli.config;
+
+import static dev.streamx.cli.util.FileUtils.createIfNotExists;
+
+import io.smallrye.config.PropertiesConfigSource;
+import java.io.File;
+import java.io.IOException;
+import java.net.URL;
+import java.nio.file.Path;
+
+public class DotStreamxGeneratedConfigSource extends PropertiesConfigSource {
+
+  public static final String CONFIG_SOURCE_NAME = "DotStreamxGeneratedConfigSource";
+  /**
+   * Value lower than classpath application.properties
+   */
+  public static final int DOT_STREAMX_GENERATED_PRIORITY = 240;
```

Comment:
I think this should be higher than classpath (build-time) application.properties but lower than `DotStreamxConfigSource`. (then priority should match range (`254`-`251`).

Reply:
Why ? I think that for dev purposes it should be possible to change the properties inside the project application.properties rather than searching for the one from streamx home

Reply:
for dev purposes you can use other sources (create `.env` or `config/application.properties`) . Values hardcoded in built version should be have lowest priority.